docs/content/docs/try-flink-kubernetes-operator/quick-start.md

```diff
@@ -99,7 +99,7 @@ flink-kubernetes-operator default 1 2022-03-09 17 (tel:12022030917):39:55.461359
 
 Once the operator is running as seen in the previous step you are ready to submit a Flink job:
 ```bash
-kubectl create -f https://raw.githubusercontent.com/apache/flink-kubernetes-operator/release-0.1/examples/basic.yaml
+kubectl create -f https://raw.githubusercontent.com/apache/flink-kubernetes-operator/release-1.0.0/examples/basic.yaml
```

Comment:
Maybe it would be better to just have `release-1.0/...`?

Reply:
Hi @gyfora, sure that looks like it works as well, I'll make that change.  Thanks!